openerp/addons/base/res/res_users.py

```diff
@@ -262,16 +262,9 @@ def _get_companies(self, cr, uid, context=None):
         return False
 
     def _get_group(self,cr, uid, context=None):
-        dataobj = self.pool.get('ir.model.data')
-        result = []
-        try:
-            dummy,group_id = dataobj.get_object_reference(cr, SUPERUSER_ID, 'base', 'group_user')
-            result.append(group_id)
-            dummy,group_id = dataobj.get_object_reference(cr, SUPERUSER_ID, 'base', 'group_partner_manager')
-            result.append(group_id)
-        except ValueError:
-            # If these groups does not exists anymore
-            pass
+        default_user_id = self.pool['ir.model.data'].xmlid_to_res_id(cr, uid, 'base.default_user')
```

Comment:
Be carefull also that some people could remove this user. The code should not crash, if default user does not exist, the default groups is void.